hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/ContractHTSSuite.java

```diff
@@ -98,6 +101,11 @@ final HapiSpec nonZeroTransfersFail() {
         final var theSecondReceiver = "somebody2";
         return defaultHapiSpec("NonZeroTransfersFail")
                 .given(
+                        snapshotMode(
+                                SnapshotMode.FUZZY_MATCH_AGAINST_HAPI_TEST_STREAMS,
+                                SnapshotMatchMode.NONDETERMINISTIC_CONSTRUCTOR_PARAMETERS,
+                                SnapshotMatchMode.NONDETERMINISTIC_FUNCTION_PARAMETERS,
+                                SnapshotMatchMode.NONDETERMINISTIC_TRANSACTION_FEES),
```

Comment:
We can just use `fuzzyMatch=true` on the `@HapiTestSpec` instead

Reply:
Yes, I tried, but sometimes it fails because transaction fees are not the same.

Reply:
@georgi-l95 In that case we should just pass the needed snapshot modes into the `defaultHapiSpec()` factory (e.g. [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/opcodes/ExtCodeSizeOperationSuite.java#L80)) and then set `@HapiTestSuite(fuzzyMatch = true)`.

Reply:
Ah, gotcha. Thank you, I've applied your suggestion, can you take a look again ?

Reply:
Done! 😁